### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MaxShylo**
- [DEMO LINK](https://MaxShylo.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://MaxShylo.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --blue-color: #00acdc;
+  --main-vertical-padding: 16px;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 0 0;
```
Comment (aliceblunt8) [inline src/style.css:18]: add padding for all card's sides and remove paddings for all inner elements

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,82 @@
...
+        </p>
+      </div>
+
+      <div class="product-card__code-container">
+        <p class="product-card__code-text">
+          Код товара: 195434
+        </p>
+      </div>
+
+      <div class="product-card__ranking-container">
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:51]: do you really need this div

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,147 @@
+/* stylelint-disable CssSyntaxError */
+/* stylelint-disable block-no-empty */
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:2]: it is not our way to hide mistakes

Comment (Bohdan-Maksymiuk) [reply src/style.css:2]: correct them

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Good job 👍 
fix comments

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**